### PR TITLE
docs: don't repeat the same doc in the README

### PR DESCRIPTION
**Pull request by spacewander**
Signed-off-by: spacewander <spacewanderlzx@gmail.com>

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by tokers — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -141,95 +141,14 @@ A/B testing, canary release, blue-green deployment, limit rate, defense against
...
-   ```
-
-   - Docker image
-
-   ```shell
-   $ git clone https://github.com/apache/apisix-docker.git
-   $ cd apisix-docker
-   $ sudo docker build -f alpine-dev/Dockerfile .
-   ```
+There are several ways to install the Apache Release version of APISIX.
```
Comment (Yiyiyimu) [inline README.md:150]: Remove since this line give no information here

File: README.md [hunk 2]
```diff
@@ -141,95 +141,14 @@ A/B testing, canary release, blue-green deployment, limit rate, defense against
...
+Please refer to [install documentation](docs/en/latest/how-to-build.md).
```
Comment (Yiyiyimu) [inline README.md:151]: We might need to move some docs from here to `how-to-build`, since `how-to-build` present less specific steps compared to here.

**Review by Yiyiyimu — COMMENTED** [review]
LGTM mostly

File: README.md (hunk 2 above)
Comment (spacewander) [reply README.md:151]: What steps?

Comment (Yiyiyimu) [reply README.md:151]: My bad, no steps need to be moved. Failed to check carefully

**Review by Yiyiyimu — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (spacewander) [reply README.md]: Updated.

**Pull request merged**